src/main/resources/application.yml

```diff
@@ -143,13 +144,13 @@ retries:
   backoff: 5000
 
 gcp:
-  project: ras-rm-sandbox
-  receiptSubscription: ras-rm-receipt
-  caseNotificationTopic: "test_topic"
-  caseNotificationSubscription: "test_subscription"
+  project: ras-rm-dev
+  receiptSubscription: sdx-receipt-sinhaa
```

Comment:
Probably wanna revert these before you merge it in

Reply:
ahh... sorry for this.. will revert it